source/_integrations/roomba.markdown

```diff
@@ -52,7 +52,7 @@ password:
   required: true
   type: string
 certificate:
-  description: Path to your certificate store.
+  description: "**(Deprecated)** Path to your certificate store."
   required: false
   type: string
   default: /etc/ssl/certs/ca-certificates.crt
```

Comment:
This should have been removed, as it is rendered useless with that PR.

Reasoning:
- New users don't have to know it, avoid confusion
- It will be in the release notes breaking changes section for existing users
- It will be logged in their instances for existing users
- Removing it now saves the maintenance burden of having documentation riddles with deprecated stuff that isn't actually in the codebase anymore (and possibly breaking configuration).